### PR TITLE
EPMRPP-39844 fixed autoscroll on launches

### PR DESCRIPTION
**Pull request by twinfreaks**

### CODE REVIEW

File: app/src/components/main/grid/gridBody/gridRow/gridRow.jsx [hunk 1]
```diff
@@ -52,25 +54,36 @@ export class GridRow extends Component {
...
     expanded: false,
+    updateHighlight: true,
   };
 
   componentDidMount() {
     this.handleAccordion();
+
+    if (this.props.isSomeRowHighlighted && this.state.updateHighlight) {
+      // eslint-disable-next-line
+      this.setState({ updateHighlight: false });
```
Comment (DiscoElevator) [inline app/src/components/main/grid/gridBody/gridRow/gridRow.jsx]: Looks like a hack. What is it for?

File: app/src/components/main/grid/gridBody/gridRow/gridRow.jsx [hunk 2]
```diff
@@ -33,6 +34,7 @@ export class GridRow extends Component {
     excludeFromSelection: PropTypes.arrayOf(PropTypes.object),
     gridRowClassName: PropTypes.string,
     level: PropTypes.number,
+    isSomeRowHighlighted: PropTypes.bool,
```
[the remark this answers is not in the document]
Comment (twinfreaks) [reply app/src/components/main/grid/gridBody/gridRow/gridRow.jsx]: It's a different value

File: app/src/components/main/grid/gridBody/gridRow/gridRow.jsx (hunk 1 above)
Comment (twinfreaks) [reply app/src/components/main/grid/gridBody/gridRow/gridRow.jsx]: In some cases, UI is not updated. So I need to trigger render here.

File: app/src/components/main/grid/gridBody/gridRow/gridRow.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (DiscoElevator) [reply app/src/components/main/grid/gridBody/gridRow/gridRow.jsx]: Looks confusing. Especially when it is set like this: `isSomeRowHighlighted={!!rowHighlightingConfig.highlightedRowId}`. I think it's better to spend some time and reuse already existing props rather that create another one with almost the same name.

**Pull request merged**